packages/leap/lib/src/leap_game.dart

```diff
@@ -41,16 +40,10 @@ class LeapGame extends FlameGame with HasTrackedComponents {
   /// and use tile size [tileSize].
   Future<void> loadWorldAndMap({
     required String tiledMapPath,
-    required int tileCameraWidth,
-    required int tileCameraHeight,
+    required CameraComponent camera,
     String prefix = 'assets/tiles/',
   }) async {
-    // Default the camera size to the bounds of the Tiled map.
-    camera = CameraComponent.withFixedResolution(
-      width: tileSize * tileCameraWidth,
-      height: tileSize * tileCameraHeight,
-      world: world,
-    );
+    camera.world = world;
```

Comment:
I don't think you need to set `world`, it's already set by `FlameGame.camera = camera` right?

Reply:
not sure, just wanted to make sure the world is added to the camera we're passing